mb_qol_seed_recording_disambiguation.user.js

```diff
@@ -129,7 +129,10 @@ function formatRecordingBareArea(area) {
     else country = country.name;
 
     state = state && state.primary_code.split('-')[1];
-    city = city && city.name;
+    // Exception for Washington D.C., it's set as a subdivision in MB, leading
+    // to comments for venues in DC to be "live, ...: <venue>, DC, USA" without
+    // the city name.
+    city = (city && city.name) || (state === 'DC' && 'Washington');
```

Comment:
If there's no city and the state isn't DC, this will be false, meaning it'll be filtered out in the next line. It could be done a bit cleaner though, but this is a "legacy" script which needs to be converted to TS at some point, so we'll clean it up there.